### PR TITLE
move social buttons in top menu so they're not hidden

### PR DESCRIPTION
**Pull request by guushoekman**

### CODE REVIEW

File: pmg/templates/_top_links.html [hunk 1]
```diff
@@ -9,4 +9,12 @@
   <li><a href="/bills">Bills</a></li>
   <li><a href="/question_replies">Questions and Replies</a></li>
   <li><a href="/calls-for-comments">Calls for Comments</a></li>
+  <div class="pull-right hidden-xs hidden-sm">
```
Comment (longhotsummer) [inline pmg/templates/_top_links.html]: This <div> should be an <li>

**Comment by guushoekman** [comment]
The padding looks random but it's to right-align the buttons with the login / user button.

The CSS is applied to `.btn` in `nav.links`. Is that alright or should these have their own class?

File: pmg/static/resources/css/style.scss [hunk 2]
```diff
@@ -111,6 +111,9 @@ nav.top-links {
 
   .btn {
     margin-top: 6px;
+    padding: 5px 10px;
+    margin-left: 9px;
+    margin-right: -5px;
```
Comment (longhotsummer) [inline pmg/static/resources/css/style.scss]: Doesn't this break any buttons in the mega menu? I'd add a `.social-buttons` class to the LI and style on `.social-buttons .btn`

**Comment by guushoekman** [comment]
Didn't break anything but yeah that makes more sense. Done.

**Comment by longhotsummer** [comment]
Looks good!

**Pull request merged**